src/components/FindMovie/FindMovie.tsx

```
@@ -16,42 +88,58 @@ export const FindMovie: React.FC = () => {
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-danger"
+              value={query}
+              className={classNames('input', { 'is-danger': error })}
+              onChange={event => {
+                setQuery(event.target.value);
+                setError('');
```

Good: error is cleared on input change (`onChange` sets `setError('')`) which satisfies the requirement to hide the error when the title changes. Also good: loading is cleared in `.finally(() => setLoading(false))`. These meet requirements 1.7 and 1.5 respectively.